src/cascadia/TerminalControl/TermControl.h

```diff
@@ -263,6 +267,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
         winrt::Windows::UI::Xaml::Controls::SwapChainPanel::LayoutUpdated_revoker _layoutUpdatedRevoker;
 
+        bool _isReadOnly{ false };

```

Comment:
```suggestion
        bool _isReadOnly;
```
nit: we generally initialize the value in the constructor.

Reply:
That's not.. strictly true. We've been back and forth on that. I think we've been moving towards constructing in the header when possible.

Reply:
_derp_, ignore me then haha